Terminal.Gui/Views/TextView.cs

```diff
@@ -1,27 +1,91 @@
+#nullable enable
+
 // TextView.cs: multi-line text editing
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Globalization;
 using System.IO;
 using System.Linq;
 using System.Runtime.CompilerServices;
 using System.Text;
+using System.Text.Json.Serialization;
 using System.Threading;
 using Terminal.Gui.Resources;
 
 namespace Terminal.Gui {
+	/// <summary>
+	/// Represents a single row/column within the <see cref="TextView"/>. Includes the glyph and the foreground/background colors.
+	/// </summary>
+	[DebuggerDisplay ("{DebuggerDisplay}")]
+	public class RuneCell : IEquatable<RuneCell> {
+		/// <summary>
+		/// The glyph to draw.
+		/// </summary>
+		[JsonConverter (typeof (RuneJsonConverter))]
+		public Rune Rune { get; set; }
+
+		/// <summary>
+		/// The foreground color to draw the glyph with.
```

Comment:
This is confusing. If it's just the `foreground color to draw with` why is the type `ColorScheme`?